build-utils/css/constants/theme/day/colors/_blue.ts

```diff
@@ -1,22 +1,24 @@
 export const blue = {
-  /** @info #E7F6FE  */
-  'blue-100': '#E7F6FE',
-  /** @info #C9E6FD  */
-  'blue-200': '#C9E6FD',
-  /** @info #A8CEFA  */
-  'blue-300': '#A3CBFA',
+  /** @info #E7F0FE  */
+  'blue-50': '#E7F0FE',
+  /** @info #DAE7FC  */
+  'blue-100': '#DAE7FC',
+  /** @info #C2D6FB  */
+  'blue-200': '#C2D6FB',
+  /** @info #98B6F8  */
+  'blue-300': '#98B6F8',
   /** @info #92B6F7  */
-  'blue-400': '#7FA9F5',
-  /** @info #7292F3  */
-  'blue-500': '#5A80F1',
-  /** @info #445EEE  */
-  'blue-600': '#445EEE',
-  /** @info #3843C2  */
-  'blue-700': '#3843C2',
-  /** @info #2E3D94  */
-  'blue-800': '#2E3D94',
-  /** @info #212F68  */
-  'blue-900': '#212F68',
-  /** @info #152141  */
-  'blue-1000': '#152141',
+  'blue-400': '#678FF3',
```

Comment:
Can you also update the jsdoc comment to match with the new color value?

Reply:
Ah yes. I thought I did but must have missed some